test/distributed/test_c10d_nccl.py

```diff
@@ -2160,16 +2186,29 @@ def test_ddp_checkpointing_twice(self):
                     self.CheckpointTwiceModule(),
                     process_group=process_group,
                     use_bucket_view=use_bucket_view,
-                    static_graph=False,
                 )
+                model = self._test_ddp_checkpointing(
```

Comment:
Shouldn't this be under its own `assertRaisesRegex` block? I don't think this line would get executed since the previous line itself would throw an exception?